python/ray/autoscaler/_private/kuberay/autoscaling_config.py

```diff
@@ -99,8 +99,8 @@ def _derive_autoscaling_config_from_ray_cr(ray_cr: Dict[str, Any]) -> Dict[str,
 
     # The autoscaler expects a global max workers field. We set it to the sum of
     # node type max workers.
-    global_max_workers = sum(
-        node_type["max_workers"] for node_type in available_node_types.values()
+    global_max_worker_nodes = sum(
+        node_type["max_worker_nodes"] for node_type in available_node_types.values()
```

Comment:
@kevin85421 I still need to add code here to make this part backwards compatible, right?

Reply:
I don't think so, but it's best to run tests if you want confirmation.